react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      topicOpen: null
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    event.target.textContent === this.state.topicOpen
+      ? this.setState({ topicOpen: null })
+      : this.setState({ topicOpen: event.target.textContent });
+  }
```

Comment:
Using the topic name for tracking what is open could be a bit dangerous, there is a chance that two topics could have the same name. Typically we would use something like an `id` to guarantee it will be unique to each item